LSP-intelephense.sublime-settings

```diff
@@ -12,6 +14,7 @@
 			"feature_selector": "source.php",
 		},
 	],
+	"auto_complete_selector": "punctuation.accessor | punctuation.definition.variable | punctuation.separator.namespace | comment.block | punctuation.definition.tag.begin",
```

Comment:
Is `comment.block` actually doing anything?
And if it does, would someone really want to trigger completions on each character in the block comments?

Reply:
Seems necessary for ![image](https://user-images.githubusercontent.com/6594915/122345646-a8ce8a80-cf7a-11eb-9bda-acd193bd7c3c.png)

Reply:
It seems like for me this also doesn't apply but I'm not sure the reason. Can't find any custom overrides...

```
>>> view.settings().get('auto_complete_selector')
'meta.tag, source - comment - string.quoted.double.block - string.quoted.single.block - string.unquoted.heredoc'
```

Reply:
Try this  `view.settings().get('auto_complete_triggers')`

Reply:
Ah, yes, it's there:

> [{'characters': '<', 'selector': 'text.html, text.xml'}, {'rhs_empty': True, 'selector': 'punctuation.accessor'}, {'selector': 'punctuation.definition.type.begin.js'}, {'characters': '$>:\\/*.<', 'selector': 'punctuation.accessor | punctuation.definition.variable | punctuation.separator.namespace | comment.block | punctuation.definition.tag.begin', 'server': 'LSP-intelephense'}]

But that means that all those characters(`$>:\/*.<`) will trigger completions in the comment blocks.

Since I don't code in PHP much, I will let you decide whether that's acceptable.

Reply:
It seems appropriate to set comments separately in ST settings

```
{
	"characters": "*",
	"selector": "comment.block",
}
```

@jfcherng  What do you think?

Reply:
I agree. `*` in comments can be treated as special case (in PHP) as it does far more than what it's for.